ConcernsCaseWork/ConcernsCaseWork.API.Tests/Integration/ConcernsCaseIntegrationTests.cs

```diff
@@ -173,6 +174,42 @@ public async Task CanCreateNewConcernCase_WithNullDivision()
 		createdCase.CaseLastUpdatedAt.Should().Be(createRequest.CreatedAt);
 	}
 
+	[Fact]
+	public async Task CanCreateNewConcernCase_WithNullRegion()
```

Comment:
Try and merge this with an existing test, (CanCreateNewConcernCase_WithNullCompaniesHouseNumber, CanCreateNewConcernCase_WithNullDivision)

If you could merge those two together into one test that would be good

A good way to do this is to have a test that checks an API with minimum fields set, that way it makes sure the api can handle just the minimum required information